app/src/Staging/java/com/simpragma/recipe/recipeapplication1/Constants1.java

```diff
@@ -0,0 +1,11 @@
+package com.simpragma.recipe.recipeapplication1;
```

Comment:
recipeapplication1 is not a good package name.

Reply:
Modified recipeapplication1 name as recipeapp